app/assets/stylesheets/book_lists.css.scss

```diff
@@ -0,0 +1,13 @@
+
+.booklist-img {
+  width: 100px;
+  padding-left: 10px;
+}
+
+.booklist-body {
+  padding-left: 10px;
+}
+
+.booklist-selected {
+  border-left: 2px solid #428bca;
```

Comment:
Color literals like `#428bca` should only be used in variable declarations; they should be referred to via variable everywhere else.